### DISCUSSION TITLE
미등록 IP에 의한 API 호출

### DISCUSSION THREAD

**Discussion by circleplatform** [commerce-api-naver/commerce-solutions#92] [묻고 답하기] [answered]
안녕하세요. 셀러박스 개발팀 이준원입니다.

저희도 아래 질문과 같이 운영 중인 App ID와 다른 App ID에서 모니터링 탐지 메일이 며칠 전부터 오고 있습니다.

![image](https://github.com/user-attachments/assets/6358cb5c-91e8-4fd2-aae0-63fede60a137)
![image](https://github.com/user-attachments/assets/edf31cea-6c0c-46c0-90df-e035b5b66d4e)

`내 솔루션 > 개발사 입점 정보` 쪽에는 IP를 입력하는 칸이 없구요.

혹시 저희 쪽에서 잘못 설정한 부분이 있을지 확인 한번 부탁드립니다.

감사합니다.

**Comment by commerce-api-naver** [comment]
@circleplatform 

안녕하세요 커머스API 권현철입니다.

말씀하신 내용에 대해 답변 드리겠습니다.

커머스API센터에 12월 11일 공지된 내용처럼 커머스솔루션 운영 정책에 변경 사항이 있었습니다.
변경 사항으로는 커머스API 호출 시 솔루션 정보에 등록하신 'API호출 IP' 이외에 다른 IP 호출로 솔루션 이용에 영향이 발생한 경우가 있어 이와 관련한 모니터링 정책을 시행하며 이는 12월 12일 커머스API센터에 사입된 개발자 계정 중 솔루션 심사를 완료(통과)한 커머스솔루션을 보유한 계정에 적용됩니다.
현재 셀러박스의 솔루션 애플리케이션에 등록된 IP와 커머스API 호출에 사용된 IP가 서로 다르게 설정되어 메일이 발송되었습니다.
IP등록 및 수정은 [커머스API센터 > 내 솔루션 > 사용중인 애플리케이션 선택 > 솔루션 애플리케이션 > API호출 IP]를 통해 가능하오니 공지 사항 및 답변 내용을 참고하시어 IP수정 부탁드립니다.

- https://notice.naver.com/notices/cac/18033? (커머스API센터)

감사합니다.

**Comment by circleplatform** [comment]
안녕하세요.
저희는 아래 이미지처럼 내 솔루션에 솔루션 정보가 없는 개발사입니다.
![image](https://github.com/user-attachments/assets/9f83f748-db35-482b-98f6-2a998056ad78)

대신 내 스토어 애플리케이션 안쪽에서 IP 3개를 등록했습니다.

이 경우에는 어떻게 하면 좋을지 확인 부탁드립니다.

감사합니다.

**Reply by commerce-api-naver** [reply, marked as answer]
@circleplatform 

안녕하세요 커머스API 권현철입니다.

'내스토어 애플리케이션'의 경우 일반 스마트스토어 판매자 분들께 제공되는 애플리케이션으로 솔루션 애플리케이션과는 다른 애플리케이션입니다.
커머스API센터의 애플리케이션, 솔루션은 로그인한 계정을 기반으로 각종 정보가 노출됩니다.
공유해주신 이미지로 보아 로그인한 계정과 실제 셀러박스, 커머스솔루션을 관리하는 커머스ID와는 서로 다른 것으로 추정됩니다.
번거로우시겠지만 커머스솔루션을 관리하는 커머스ID 계정으로 로그인 후 솔루션의 API호출IP 수정 가능하실까요?

감사합니다.

**Reply by circleplatform** [reply]
안녕하세요.

말씀 주신 방법으로 방법을 찾아서 진행했는데, 심사신청 페이지 URL과 Redirect URL이 다르다고해서 저장이 안되네요..ㅠㅠ
혹시 이거 편집할 수 있는 방법이 있을까요? www를 빼고 싶습니다.

![image](https://github.com/user-attachments/assets/11edd5d7-6402-4fef-8fc3-445247cb15dc)

**Reply by commerce-api-naver** [reply]
@circleplatform 

안녕하세요 커머스API 권현철입니다.

커머스솔루션 정책에 따라 심사완료 후 일부 솔루션 정보는 수정이 불가능합니다.
다만, '셀러박스'의 경우 커머스API센터 내 솔루션 관리 기능 오픈 이전 초기 연동 지원 솔루션사로 관련 정보를 네이버에서 관리를 해줬기에 예외적으로 동일 URL에 대해서 `www.`만 제거하는 방향으로 지원해드렸습니다. 번거로우시겠지만 API호출 IP 재수정 시도 가능하실까요?
추가로 이러한 예외적인 상황 외에는 정보 수정을 지원하지 않으며 변경이 가능한 솔루션 정보는 커머스API센터에서 직접 수정 하는 것을 권장드립니다.

감사합니다.

**Reply by circleplatform** [reply]
안녕하세요.

도와주신 덕분에 IP 세개 등록을 완료했습니다.

감사합니다.
